internal/buf/bufimage/bufimagebuild/bufimagebuildtesting/crashers/e55d098c6ca004c39f85be18610ecad5da52b268.output

```diff
@@ -0,0 +1,12 @@
+panic: buf has file annotations but protoc has no error: [/var/folders/gt/cvjl_x695bg3n1p617s0gs6c0000gn/T/buffuzz136699593/000.proto:1:1:syntax error: unexpected $unk]
+
+goroutine 1 [running]:
+github.com/bufbuild/buf/internal/buf/bufimage/bufimagebuild/bufimagebuildtesting.(*fuzzResult).panicOrN(0xc000270f00, 0x193f220, 0xc00002c108, 0x13)
+	/Users/wroden/repos/WillAbides/buf/internal/buf/bufimage/bufimagebuild/bufimagebuildtesting/bufimagebuildtesting.go:205 +0x16c
```

Comment:
All of these extra files - do we have to check them in? If we do, we for sure can't check in `/Users/wroden`

Reply:
The *.output and *.quoted crasher files could probably be `.gitignored` TBH. As long as we store the input for future runs we can reproduce the bugs found.

Reply:
Good idea with .gitignore. I'll do that.